R/local-moran-bv-impl.R

```diff
@@ -36,7 +37,7 @@ local_moran_bv_perm_impl <- function(x, y, listw) {
 #' Local Bivariate Moran's I spdep implementation
 #' @inheritParams moran_bv_impl
 #' @keywords internal
-local_moran_bv_impl <- function(x, y, listw, nsim) {
+local_moran_bv_impl <- function(x, y, nb, listw, nsim) {
```

Comment:
Same as above, remove the `nb` argument.

Reply:
Similarly, see line 47 where the object `obs` (short for the observed value) should be modified to use the new syntax so 

```r
obs <- local_moran_bv_calc(x, y, nb, wt)
```

Reply:
Thank you so much for your kind feedback. I will update these and revert.

Reply:
I have made the updates. Please let me know if I got it right. Thank you for the opportunity